tests/native.js

```diff
@@ -163,3 +163,56 @@ Native["com/sun/midp/i18n/TestResourceConstants.setLanguage.(Ljava/lang/String;)
 // Many tests create FileConnection objects to files with the "/" root,
 // so add it to the list of valid roots.
 MIDP.fsRoots.push("/");
+
+var NO_SECURITY = typeof netscape !== "undefined" && netscape.security.PrivilegeManager;
```

Comment:
Indeed. In b60f0b3, I move it back to benchmark.js, refactoring *sampleMemory* in that file so that *MemorySampler.sampleMemory* can use it from tests/native.js. I hear @brendandahl is changing the way we sample memory, so that function might change further, but in the meantime this means we aren't duplicating its functionality.

Note that you now have to `make BENCHMARK=1` to see memory results. Without *BENCHMARK*, you'll only see timings.